Ginger/GingerCoreNET/WizardLib/WizardBase.cs

```diff
@@ -132,7 +132,7 @@ public void ProcessEnded()
         //    }
         //}
 
-                
+        public bool PrevEnabled { get; set; } = true;
```

Comment:
i see that you want to prevent going back on once user got to last page so, change the property name to be "DisableBackBtnOnLastPage"